### ISSUE TITLE
Force-loaded chunks not staying loaded

### ISSUE THREAD

**Issue by ukmcplyr** [FTBTeam/FTB-Modpack-Issues#2310]
### Modpack

FTB Presents Direwolf20 1.19

### Modpack version

1.0.4

### Log Files

N/A

### Describe the bug

Chunks marked as forced-loaded in FTB Chunks don't appear to stay loaded.

Tested on a server only.

### Steps to reproduce

On a server:
1. Place down a vanilla furnace
2. Mark containing chunk as force-loaded in FTB Chunks
3. Place a stack of cobble and enough fuel to smelt it in the furnace
4. Disconnect from server
5. Return later and check furnace's progress

### Expected behaviour

Force-loaded chunks remain loaded

**Comment by DeadZoneGaming** [comment]
I am having this issue in my single player world.

**Comment by benkravitz10** [comment]
I am _also_ having this issue in single player

**Comment by duragizer** [comment]
same single player world, seems to work if i unclaim and reclaim chunks and chunk load them each game load it is a hassle though

**Comment by nightred** [comment]
Confirmed at several player bases

**Comment by SSkizz** [comment]
I am having this same issue on a dedicated server as well. Both FTB Chunks and Flux Networks items do not properly force load the chunk.

**Comment by ukmcplyr** [comment]
I'm seeing a different kind of chunkloading strangeness now.

Before I couldn't get any chunks to stay loaded regardless of dimension. Now, I'd managed to somehow get chunkloading working in the nether after I'd deleted the nether dim from my testing server for reasons unrelated.

I also used FTB Chunks' mouse wheel "adjust force-load expiry" functionality to flip between timed expiry and no expiry for the chunk in the nether. With nether chunks staying loaded, I observed force-loaded chunks staying loaded in the overworld, too.

Whether either of these kicked FTB Chunks into life or not, though, I don't know!?

Curiously, if I restart the server chunkloading remains working in the overworld, but not in the nether...

I had a load of chunks marked as force-loaded but I've reduced that down to just a few; here's [latest.log](https://pste.ch/buvizurowe.sql) and [debug.log](https://pste.ch/hoqapafoqa.md) from post-restart in case they're helpful.
